packages/app/src/Login/MultiAccountStore.ts

```diff
@@ -217,6 +218,23 @@ export class MultiAccountStore extends ExternalStore<LoginSession> {
     return { ...s };
   }
 
+  async #loadIrisKeyIfExists() {
+    try {
+      const irisKeyJSON = window.localStorage.getItem('iris.myKey');
+      if (irisKeyJSON) {
+        const irisKeyObj = JSON.parse(irisKeyJSON);
+        if (irisKeyObj.priv) {
+          const privateKey = await PinEncrypted.create(irisKeyObj.priv, '1234');
```

Comment:
Some people dislike the pin feature, im planning to make it optional, if you want to store an unencrypted in you can make another class which just has `get value()` then it should work the same, and i can also implement this later in the UX so people can stop complaining about it